### ISSUE TITLE
Implement consistent LA boundaries

### ISSUE THREAD

**Issue by Antonio-John** [ONSdigital/SDG_11.2.1#154]
This ticket ensures that consistent LA boundaries are used within the calculations. 
This comes from a discussion with the geo spatial team documented [here ](https://github.com/ONSdigital/SDG_11.2.1/issues/152)

The latest available population estimates need to be used in the calculations. This is currently the mid-2020 for the 2011 Output Areas found [here:](https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/bulletins/annualsmallareapopulationestimates/mid2020/relateddata)

These need to be mapped to LAD boundaries for 2020, these boundaries can be found [here](https://geoportal.statistics.gov.uk/datasets/local-authority-districts-december-2020-uk-bfe/explore?location=55.339986%2C-3.313872%2C6.43):

Where the UK LA file is, this needs to be adjusted to use the 2020 boundaries. The file will need to be saved into the data folder and the config file adjusted to the name of this new file. The value in the config for uk_la_shp_file will need to match the 2020 boundaries file which has just been downloaded.
```
uk_la_file_conf = config['uk_la_shp_file']
full_path_uk_la=os.path.join(os.getcwd(), "data", "LA_shp", uk_la_file_conf)
uk_la_file=di.geo_df_from_geospatialfile(path_to_file=full_path_uk_la)
```

Similarly the population files will need to be saved into the data/population_estimates/ file, then the config variable ```calculation_year``` will need to be adjusted to 2020.

```
pop_year = config["calculation_year"]
# Get list of all pop_estimate files for target year
pop_files = os.listdir(os.path.join(
                                    "data/population_estimates",
                                    str(pop_year)
                                    )
                       )
```

For both the population and LAD boundaries, not sure if the data is in the same format as previously, this will need to be checked and QA's when new data is integrated.

**Comment by jwestw** [comment]
Good start. Shall we distill down some clear requirements from the discussion so that the developer knows exactly what to create to get this ticket done?

**Comment by Antonio-John** [comment]
Dev Requirements:
- Change the LA boundaries to the 2020 version.
- Change the population estimates to be 2020 version.

More info 
The latest available population estimates need to be used in the calculations. This is currently the mid-2020 for the 2011 Output Areas found [here:](https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/bulletins/annualsmallareapopulationestimates/mid2020/relateddata)

These need to be mapped to LAD boundaries for 2020, these boundaries can be found [here](https://geoportal.statistics.gov.uk/datasets/local-authority-districts-december-2020-uk-bfe/explore?location=55.339986%2C-3.313872%2C6.43):

**Comment by Antonio-John** [comment]
What i've done:
- Changed the config file to reflect only one year under ```calculation_year```
- Added in folders for population estimates and la boundaries for different years.
- Changed ```get_whole_nation_pop``` so that it can deal with passing in both 2019 and 2020 datasets. Also saves these as separate feather files for 2019 & 2020
- Runs perfectly for 2020.

To Do:
-Can't seem to find a good lookupfile between OA and LA for 2019. Have downloaded a postcode one but it's too big and the column names are ladnm instead of lad19m as expected.

**Comment by Antonio-John** [comment]
Hi @MusaChirikeni

I have a lookupfile for between OA11NM and LA20NM [here](https://geoportal.statistics.gov.uk/datasets/output-area-to-lower-layer-super-output-area-to-middle-layer-super-output-area-to-local-authority-district-december-2020-lookup-in-england-and-wales/explore)

I am trying to find the equivalent lookupfile between OA11NM and other years e.g LAD19NM. I can't seem to find this file. Do you know if it exists and if so where I could find it please?

Many thanks,

Antonio

**Comment by MusaChirikeni** [comment]
Hi Antonio,

You should be able to get these from the link below:
https://geoportal.statistics.gov.uk/search?collection=Dataset&sort=name&tags=all(LUP_OA_WD_LAD)

**Comment by Antonio-John** [comment]
That's great, thanks! @MusaChirikeni

**Comment by Antonio-John** [comment]
Have used filezilla with login details from Mark to upload the data files downloaded for this ticket.
This includes having sub directories with years in the following folders:
data/population_estimates/
data/LA_shp/
data/oa_la_mapping/

**Comment by james-westwood** [comment]
Closed due to merging of #167
